### ISSUE TITLE
Matching validation images to Imagenet validation annotations

### ISSUE THREAD

**Issue by hhybang** [MadryLab/backgrounds_challenge#10]
Hello,

I am trying to get the bounding boxes for the validation images of Imagenet9 and am having trouble matching them with Imagenet's validation set annotations due to the differences in how the files are named. 

The Imagenet's validation images / annotations follow the naming convention of "ILSVRC2012_val_…." whereas the validation images for Imagenet9 seem to follow the convention of "n___" which is the naming convention of Imagenet's training set. 

Could I get help in getting the annotations for the validation images? 
Thank you!!
